ivy/functional/backends/tensorflow/experimental/linear_algebra.py

```diff
@@ -59,9 +59,18 @@ def matrix_exp(
 
 
 def eig(
-    x: Union[tf.Tensor],
+    x: Union[tf.Tensor, tf.Variable],
```

Comment:
I don't think you need to change this line, as `eig` only expects Tensor as input as stated here: https://www.tensorflow.org/api_docs/python/tf/linalg/eig

Reply:
Thanks for reviewing, I made the necessary changes.